src/nnet3bin/nnet3-get-egs-dense-targets.cc

```diff
@@ -176,16 +170,22 @@ int main(int argc, char *argv[]) {
 
 
     bool compress = true;
-    int32 num_targets = -1, length_tolerance = 100, online_ivector_period = 1;
+    int32 num_targets = -1, length_tolerance = 100,
+        targets_length_tolerance = 2,  
+        online_ivector_period = 1;
+
     ExampleGenerationConfig eg_config;  // controls num-frames,
                                         // left/right-context, etc.
 
     std::string online_ivector_rspecifier;
+
     ParseOptions po(usage);
 
-    eg_config.Register(&po);
-    po.Register("compress", &compress, "If true, write egs in "
-                "compressed format.");
+    po.Register("compress", &compress, "If true, write egs with input features "
+                "in compressed format (recommended).  Update: this is now "
```

Comment:
Replace 'Update: this is now only' with 'This is only'.  We don't document the changes to options, except if likely to break things, because the average user won't be aware of, and won't care about, the history.